doc/electrs-rest-btcd-1.0.0.yaml

```diff
@@ -0,0 +1,936 @@
+      description: Returns up to 50 transactions (No Paging)
+      parameters:
+      - name: hash
+        in: path
+        required: true
+        schema:
+          type: string
+      responses:
+        '200':
+          description: Transactions
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Transaction'
+  /address/{address}/utxo:
+    get:
+      tags:
+        - address
+      summary: Get the list of unspent transaction outputs associated with the address
+      description: |
+          Elements based chains have a different repsponse structure.
+      parameters:
+        - name: address
+          in: path
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: UTXOs
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Utxo'
+  /scripthash/{hash}/utxo:
+    get:
+      tags:
+        - scripthash
+      summary: Get the list of unspent transaction outputs associated with the address
+      description: |
+          Elements based chains have a different repsponse structure.
+      parameters:
+        - name: hash
+          in: path
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: UTXOs
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Utxo'
+  /tx/{txid}:
+    get:
+      tags:
+        - tx
+      summary: Returns information about the transaction
+      parameters:
+        - name: txid
+          in: path
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Transaction data
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Transaction'
+  /tx/{txid}/status:
+    get:
+      tags:
+        - tx
+      summary: Returns transaction confirmation status by hash
+      parameters:
+        - name: txid
+          in: path
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Transaction status
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/TxStatus'
+  /tx/{hash}/merkleblock-proof:
+    get:
+      tags:
+        - tx
+      summary: Merkle Inclusing Proof (Bitcoind merkleblock)
+      description: |
+          Returns a Merkle inclusion proof for the transaction using   
+          [bitcoind's merkelblock](https://bitcoin.org/en/glossary/merkle-block) format.  
```

Comment:
link is 404

Reply:
it appears that the entire glossary section is gone looking into alternatives.

Reply:
found the right docs: https://btcinformation.org/en/glossary/merkle-block